hw1_mzw7af_vl4kz.py

```diff
@@ -57,6 +57,30 @@ def main():
     facts = {}
     rules = []
 
+    for line in sys.stdin:
+        argArray = line.strip().split(" ")
 
+        if argArray[0].lower() == "teach":
```

Comment:
Instead of argArray[0].lower for every if statement, why not just do it once and store it in a var?